### PR TITLE
new option to send data to custom server URL with POST method

### PR DESCRIPTION
**Pull request by eneiluj**
I also added the french translations.

This time i adapted the tests...

### CODE REVIEW

**Comment by mendhak** [comment]
That doesn't look so bad thanks man.  OK let me play with this, be patient with me.  Unfortunately your translations will be overwritten from crowdin I'll attempt to add them in there.

**Comment by eneiluj** [comment]
Thanks for the.....compliment.

I'm patient. I can already use this feature myself anyway ;)

I'll translate on crowdin when the new strings will appear.

File: gpslogger/src/main/java/com/mendhak/gpslogger/loggers/customurl/CustomUrlJob.java [hunk 1]
```diff
@@ -75,7 +78,35 @@ public Request authenticate(Route route, Response response) throws IOException {
...
+            if (logUrl.contains("?")) {
+                urlSplit = logUrl.split("\\?");
+                if (urlSplit.length == 2) {
+                    baseUrl = urlSplit[0];
+                    paramSplit = urlSplit[1].split("\\&");
+                    for (int i = 0; i < paramSplit.length; i++) {
+                        if (paramSplit[i].contains("=")) {
+                            String[] oneParamSplit = paramSplit[i].split("=");
+                            if (oneParamSplit.length == 2) {
+                                bodyBuilder.add(URLEncoder.encode(oneParamSplit[0], "UTF-8"), URLEncoder.encode(oneParamSplit[1], "UTF-8"));
```
Comment (mendhak) [inline gpslogger/src/main/java/com/mendhak/gpslogger/loggers/customurl/CustomUrlJob.java]: Testing locally at the moment, I removed the `URLEncoder.encode` from here as it was causing double encoding.  Will push later.

**Comment by mendhak** [comment]
OK thanks now it is merged , bit of refactor and some tests.  Also changed the summary in the  preference, visible on crowdin now

**Comment by mendhak** [comment]
v89 released to market

**Pull request merged**